frontend/src/metabase/admin/settings/components/SettingsUpdatesForm/SettingsUpdatesForm.tsx

```diff
@@ -0,0 +1,80 @@
+import cx from "classnames";
+
+import { UpsellHostingUpdates } from "metabase/admin/upsells";
+import { useSetting } from "metabase/common/hooks";
+import CS from "metabase/css/core/index.css";
+import { Flex } from "metabase/ui";
+import type { SettingDefinition, UpdateChannel } from "metabase-types/api";
+
+import { SettingsSetting } from "../SettingsSetting";
+
+import { VersionUpdateNotice } from "./VersionUpdateNotice/VersionUpdateNotice";
+
+const updateChannelSetting = {
+  key: "update-channel",
+  display_name: "Update Channel",
+  type: "select",
+  description:
+    "Select an update channel to customize which Metabase releases you receive notifications for.",
+  defaultValue: "latest",
+  options: [
+    { name: "Stable", value: "latest" },
+    { name: "Beta", value: "beta" },
+    { name: "Nightly", value: "nightly" },
+  ],
+};
```

Comment:
@jeff-bruemmer , would appreciate your eyes on this copy. this also probably needs some documentation.

see our [internal documentation](https://www.notion.so/metabase/Release-Channel-Tagging-437bdfa7b0c54579badd058a84a64c85) on this: